app/views/pages/how-platformos-works/platformos-workflow/development-workflow.liquid

```diff
@@ -61,6 +55,22 @@ You can set up different types of Instances (staging/production) through the Par
 
 Based on the project’s size, scope, or requirements, stages from the above workflow can be removed, or additional stages can be added. For example, a testing site before staging can be added to separate testing from QA and UAT.
 
+### Development Workflow  
+
+You develop your site (write code) in your **codebase in your local environment**. By syncing to your developer Instance you check your changes in real time. The `pos-cli sync` command enables sync mode: it checks what's different between the server and the local environment, and sends files after they have been changed on local. It only sends files that have changed after sync was started. It feels like working on localhost. It is recommended to use sync only for staging environments, as pushing changes immediately to production can be dangerous.
```

Comment:
> it checks what's different between the server and the local environment,

Actually it doesnt checks anything on the server. It just observes local changes and sends files that have been changed after sync has been initiated.

Reply:
Thank you! I have edited it, and will edit in other parts of the documentation, too.